test/global.setup.ts

```diff
@@ -4,7 +4,7 @@ export default () => {
   process.env.COOKIE_SECRET = "123456789012345678901234";
   process.env.CACHE_SERVER = "cache_server";
   process.env.SHOW_SERVICE_OFFLINE_PAGE = "false";
-  process.env.LOG_LEVEL = "error";
+  process.env.LOG_LEVEL = "info";
```

Comment:
did you mean to keep the tests logging at info level or did you just do it to debug?